temperature_converter/src/Main.gren

```diff
@@ -24,13 +25,14 @@ type alias Model =
 
 
 init : Model
-init = 
+init =
     { celsius = ""
     , fahrenheit = ""
     }
 
 
---- UPDATE
+
+-- - UPDATE
```

Comment:
This is interesting. How does the space get in there?

Reply:
Ah, it tries to insert a leading space in the comment if there isn't one, but I guess that doesn't make sense if it starts with `-`!  I'll have to check how I did it in elm-format.